package.json

```diff
@@ -181,6 +183,7 @@
     "physical-cpu-count": "^2.0.0",
     "postcss-colorfix": "^0.0.4",
     "postcss-loader": "3.0.0",
+    "postcss-nested": "^4.2.1",
```

Comment:
is this addition necessary? Can we leverage less to do the same thing? i am not good with computer

Reply:
In theory we could use less to do pre-processing before we pass things along to get module-ified, but my impression was that we are generally not thrilled with the idea of continuing to use a heavy pre-processor "language" like less/sass going forward at all vs. just using tiny postcss transforms as needed. Perhaps that's something we need to address in a larger discussion.

Reply:
So far I've taken the approach that we'll have 1) the 'legacy' set of non-module CSS that's usually written in `.less` files, and 2) the new set of module-ified CSS that only uses limited PostCSS transformations (nesting being probably the heaviest I'd want to drop in)